app/lib/forem_stats_drivers/datadog_driver.rb

```diff
@@ -9,9 +9,16 @@ class DatadogDriver
         c.tracer priority_sampling: true
         c.use :elasticsearch
         c.use :sidekiq
-        c.use :redis
+        c.use :redis, service_name: 'redis', describes: { url: ENV['REDIS_URL'] }
```

Comment:
FYI we are a double quote shop 😉

Reply:
Just noticed the Rubocop error, too, and fixed it. :joy: Just wait until you see some of my other code, though!

<img width="195" alt="Method definition without parentheses because I'm a rebel" src="https://user-images.githubusercontent.com/108205/110039914-f565c100-7d0f-11eb-98a6-7ace46c38376.png">

Seriously, though, I'm a fan of using lint rules that Rubocop can autocorrect. I'm too lazy to hit the shift key sometimes. 😄